cmd/daprd/options/options.go

```diff
@@ -56,6 +56,7 @@ type Options struct {
 	DaprInternalGRPCPort         string
 	DaprPublicPort               string
 	AppPort                      string
+	AppHealthPort                string
```

Comment:
We must make this is a pointer as we need to determine the different between being set or not.